src/wp-includes/class-walker-nav-menu.php

```diff
@@ -39,6 +39,30 @@ class Walker_Nav_Menu extends Walker {
 		'id'     => 'db_id',
 	);
 
+	/**
+	 * The URL to the privacy policy page.
+	 *
+	 * @since 6.7.2
+	 * @var string
+	 */
+	private static $privacy_policy_url = null;
+
+	/**
+	 * Get the URL to the privacy policy page.
+	 * This method caches the URL to avoid multiple database queries and shared the URL between all instances of the class.
+	 * Single query is performed on the first call to this method.
+	 *
+	 * @since 6.7.2
```

Comment:
This is not something for a point release, as this is not a bug introduced in 6.7. (It was added in 6.2)

Let's change this to 6.8.0